### PR TITLE
issue of liner.vue fixed

### PR DESCRIPTION
**Pull request by farzad2919**
### Request type :question:

- [x] :bug: Bug ( fixed or found bug )
- [ ] :new: New feature ( make new feature or add some things new in project )
- [ ] :star2: Config ( change config files or install )
- [ ] :rocket: Performance ( when you do something that can improve performance )
- [ ] :books: Document ( when code has documentation or document change )
- [ ] ✏️ Update (when some lines have to get updates, no bug, no new feature!)

### Description :books:

- What did you do

### How to use ⚒️

- What is necessary to do

### CODE REVIEW

**Comment by farzad2919** [comment]
issue of "liner.vue" component fixed , but i did not use this component in new page.

**Comment by farzad2919** [comment]
سلام.عذر میخوام دیر شد سیستمم به مشکل خورده هنوز نتونستم اصلاحات رو انجام
بدم.همچنان درگیر درست کردن سیستم هستم

File: components/Input/Text/Area/Liner.vue [hunk 1]
```diff
@@ -34,14 +37,18 @@ const props = defineProps({
...
-
 function line_counter(event) {
   lines.value = event.target.value.split("\n").length;
-  event.target.style.height = "100%";
-  //   event.target.style.backgroundColor = "green";
-  console.log(event.target.style);
+  // event.target.style.height = "100%";
+  if (lines.value > 7) {
+    numberCardDiv.value.style.height = "1000px";
+    textAreaDiv.value.style.height = "1000px";
```
Comment (babak-x) [inline components/Input/Text/Area/Liner.vue:48]: Seriously?

image

**Review by babak-x — CHANGES_REQUESTED** [review]
Actualy, you didn't fix it! You hide it.

**Pull request closed without merging**